packages/runtime/test/start/handle-allocated-port.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+const assert = require('node:assert')
+const { platformaticRuntime } = require('../..')
+const { join } = require('node:path')
+const { test } = require('node:test')
+const { loadConfig } = require('@platformatic/config')
+const fixturesDir = join(__dirname, '..', '..', 'fixtures')
+const { buildRuntime } = require('../../lib/start')
+
+test('startes with port +1 when current port is allocated', async () => {
+  const configFile = join(fixturesDir, 'configs', 'monorepo-port-3000.json')
+  const config = await loadConfig({}, ['-c', configFile], platformaticRuntime)
+  const app = await buildRuntime(config.configManager)
+  const res = await app.start()
+  assert.strictEqual('Hello', res)
```

Comment:
How does this assert this was started in port+1?

Reply:
I am not finished with the test yet, later i'll continue working on it. Is there any config file in the fixtures directory, to start a service with a fixed port ?

Reply:
Thanks, i think this will be a good approach :+1: